kinetic/doc-build.yaml

```diff
@@ -0,0 +1,57 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+canonical_base_url: http://54.183.65.232/docs
+documentation_type: rosdoc_lite
+jenkins_job_priority: 90
```

Comment:
The priorities are not identical across distros. For several job types the newer distros get a higher priority (which is lower numeric value). That should be the same for Kinetic.